src/NServiceBus.Extensions.IntegrationTesting/NsbActivityNames.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace NServiceBus.Extensions.IntegrationTesting
+{
+    internal static class NsbActivityNames
```

Comment:
It looks like these fields are unused. Can you remove them?
- OutgoingEventActivityName
- SubscribeActivityName
- UnsubscribeActivityName

Reply:
Done!  Thanks!